### DISCUSSION TITLE
rc.9 ERR_MODULE_NOT_FOUND with Yarn

### DISCUSSION THREAD

**Discussion by Mabi19** [nuxt/nuxt#16884] [Questions]
Something related to Vite started erroring out when I updated to rc.9. This issue was already brought up in a comment thread on the rc.9 release post, but the solution proposed there is not applicable for me (I'm using Yarn 1).

I'd make an issue if I had a small reproduction, but I don't have the time to make that right now.

Here are my dependencies, since they are probably related:
```
"devDependencies": {
"@types/formidable": "^2.0.5",
"@types/latinize": "^0.2.15",
"@types/uuid": "^8.3.4",
"nuxt": "^3.0.0-rc.9",
"sass": "^1.45.0"
},
"dependencies": {
"@tiptap/core": "^2.0.0-beta.182",
"@tiptap/extension-blockquote": "^2.0.0-beta.29",
"@tiptap/extension-bold": "^2.0.0-beta.28",
"@tiptap/extension-bullet-list": "^2.0.0-beta.29",
"@tiptap/extension-document": "^2.0.0-beta.17",
"@tiptap/extension-hard-break": "^2.0.0-beta.33",
"@tiptap/extension-heading": "^2.0.0-beta.29",
"@tiptap/extension-history": "^2.0.0-beta.26",
"@tiptap/extension-italic": "^2.0.0-beta.28",
"@tiptap/extension-list-item": "^2.0.0-beta.23",
"@tiptap/extension-ordered-list": "^2.0.0-beta.30",
"@tiptap/extension-paragraph": "^2.0.0-beta.26",
"@tiptap/extension-strike": "^2.0.0-beta.29",
"@tiptap/extension-subscript": "^2.0.0-beta.13",
"@tiptap/extension-superscript": "^2.0.0-beta.13",
"@tiptap/extension-text": "^2.0.0-beta.17",
"@tiptap/extension-typography": "^2.0.0-beta.22",
"@tiptap/extension-underline": "^2.0.0-beta.25",
"@tiptap/vue-3": "^2.0.0-beta.96",
"formidable": "^2.0.1",
"hash-wasm": "^4.9.0",
"latinize": "^0.5.0",
"postgres": "^3.2.4",
"uuid": "^8.3.2",
"zeed-dom": "^0.9.25"
},
```
Most of them are only used on the server side, with the exception of Tiptap.

**Comment by TheAlexLichter** [comment]
Hard to tell without the actual reproduction. 🙁

**Comment by Mabi19** [comment]
Diagnosing this issue further has been really hard (mostly due to several unrelated distractions and the extremely unhelpful error message). It ended up being a seemingly inconspicous, easily missed import from `#build/components` (because of a workaround to [this bug](https://github.com/nuxt/framework/issues/6631)).
